04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -77,43 +88,34 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/)
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
 * [Lesson 07- Extract NDVI Summary Values from a Raster Time Series]({{ site.baseurl}}/R/Extract-NDVI-From-Rasters-In-R/)
-</div>
 
-#About Raster Bands
+###Sources of Additional Information
+<a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank">
+Read more about the `raster` package in R.</a>
+</div>
 
-As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}NEON-R-Spatial-Raster/R/Introduction-to-Raster-Data-In-R), a raster can
-contain 1 or more bands.
+#About Raster Bands in R 
+As discussed in the [Intro to Raster Data Lesson]( {{ base.url }} }}NEON-R-Spatial-Raster/R/Introduction-to-Raster-Data-In-R), 
+a raster can contain 1 or more bands.
 
 <figure>
     <a href="{{ site.baseurl }}/images/raster_timeseries/single_multi_raster.png">
     <img src="{{ site.baseurl }}/images/raster_timeseries/single_multi_raster.png"></a>
-    <figcaption>A raster dataset can contain one or more bands. We can use the raster 
-    function to import one single band from a single OR multi-band raster.</figcaption>
+    <figcaption>A raster dataset can contain one or more bands. We can use the
+    raster function to import one single band from a single OR multi-band
+    raster.  Image Source: NEON, Inc.</figcaption>
 </figure>
 
-To work with multi-band rasters, we need to change how 
-we import and plot our data in several ways. 
+To work with multi-band rasters in R, we need to change how we import and plot
+our data in several ways. 
 
 * To import multi band raster data we will use the `stack()` function.
 * If our multi-band data are imagery that we wish to composite, we can use
 `plotRGB()` (instead of `plot`) to plot a 3 band raster image.
 
```

Comment:
Okay.  But then do we change it from "Want to learn more".  As previously written I saw it as additional information, not information that was integral to the lesson.